### DISCUSSION TITLE
account mark flagged

### DISCUSSION THREAD

**Discussion by hacanand** [community/community#110885] [New to GitHub] [answered]
@hacanand My account (@hacanand) mark flagged. I raise a ticket to the Github team. I did not get reply even after 15 days, It stopped my development work. Please suggest some relevant step so that it can be resolved within a time.
@github-staff @davidstaheli @zackfern @mgriffin @davidstaheli @grandmas-favorite @ghostinhershell @taylorcuilty @support

**Comment by hacanand** [comment, marked as answer]
please reply with relevant solution

**Comment by ATarasenko19** [comment]
We have the same situation. 2 weeks the work almost stopped. Awful (An account that was flagged @NikitaGolitsyn)

**Comment by MDP43140** [comment]
Happened to me too @mdp43140. Mine got flagged after doing a commit in GitHub desktop (linux flatpak version), and changing email provider from Gmail to ProtonMail, it doesn't happen before, i used GitHub desktop for years without any issues, until now 😭

**Comment by OneOfTheMicahs** [comment]
Same situation for me—unknowingly flagged, which is preventing me from contributing to a project I joined. This is ridiculous.

**Comment by Cobeine** [comment]
Same here, 2 weeks and still 0 answers

**Comment by DennisMwongera** [comment]
@hacanand  and  @ATarasenko19  were your issues resolved. Because I have been in a similar situation for now 3 months.

**Comment by kubikaugustyn** [comment]
I have the same issue. My account has been marked as "Spammy", I cannot fork repos nor comment and my profile was hidden, hiding even my public repos that were being in use by the public. Got no answers to my support ticket and thus I cannot work as usual.

**Comment by germanztz** [comment]
I have the same issue, account flagged, cannot search for code and my repos are made private. please can you help

**Reply by HimanshuMohanty-Git24** [reply]
Hello @germanztz  How did you solve this problem ? I have the same issue and others can't view my account
![image](https://github.com/community/community/assets/94133298/86d66357-38a1-46e7-9299-6a20f44eabf8)

**Comment by AmineRACHID** [comment]
I have the same issue, account flagged, cannot search for code and my repos are made private. please can you help

**Comment by cyx200902** [comment]
I have also encountered the same problem, possibly due to my unstable network environment. I submitted my application two weeks ago but still haven't received any response.

**Comment by RaihanArdianata** [comment]
My team members are consistently receiving a 'Not Found' error when trying to access my GitHub profile

**Comment by Arpitgarg07** [comment]
No, you can't. Creating two accounts from one device is banned.
